site/en/blog/new-in-chrome-104/index.md

```diff
@@ -0,0 +1,210 @@
+---
+title: New in Chrome 104
+description: >
+  Chrome 104 is rolling out now! With region capture, you can now specify a
+  crop area when using getDisplayMedia to capture the current tab. Media
+  query syntax can be written using mathematical comparison operators.
+  Shared Element Transitions start a new origin trial. And there's plenty
```

Comment:
```suggestion
  Shared Element Transitions starts an origin trial. And there's plenty
```

Reply:
This subject-verb agreement is a tricky one. It's not the morphology of 'transitions' that governs this verb. Because "Shared Element Transitions" is a formal name, you treat the whole as a noun, not as a noun with two adjectives. "Shared Element Transitions" is a singular noun referring to the feature, so it takes a plural verb.

Reply:
>'a new' to 'an'

This is its first origin trial.